app/helpers/audits_helper.rb

```diff
@@ -115,34 +116,14 @@ def audit_action_name(audit)
 
   def audit_user(audit)
     return if audit.username.nil?
-    login = audit.user.login rescue nil # aliasing the user method sometimes yields strings
+    login = audit.user_login
```

Comment:
why remove this? AFAIR this allow to handle cases where the user was deleted but still visible in audits?